ratis-server-api/src/main/java/org/apache/ratis/server/protocol/RaftServerProtocol.java

```diff
@@ -34,4 +36,6 @@
   AppendEntriesReplyProto appendEntries(AppendEntriesRequestProto request) throws IOException;
 
   InstallSnapshotReplyProto installSnapshot(InstallSnapshotRequestProto request) throws IOException;
+
+  TimeoutNowReplyProto timeoutNow(TimeoutNowRequestProto request) throws IOException;
```

Comment:
Let's call this new method startLeaderElection.

Reply:
@szetszwo Do we also rename TimeoutNowRequestProto to StartLeaderElectionRequestProto and TimeoutNowReplyProto to StartLeaderElectionReplyProto ?

Reply:
> Do we also rename all the string TimeoutNow to StartLeaderElection ? ...

Yes, thanks.  "Timeout" is too general.